docs/source/analyze/certificates.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Certificates
+description: Perform analysis and checks on Certificates found in the cluster
+---
+
+The Certificates analyzer serves to alert users when a specific certificate is either invalid or nearing its expiration date.
+This analyzer's outcome `when` clause compares the condition specified with the resources present on each or all certificates.
```

Comment:
>on each or all certificates.

Could this be simplified? I wasn't sure what "each or all" meant. Maybe just "resources present on the certificates"?

Reply:
Thank you! Agree, resources present on the certificates is more appropriate.